package.json

```diff
@@ -103,7 +103,7 @@
     "react-native-inappbrowser-reborn": "^3.0.0",
     "react-native-linear-gradient": "^2.5.4",
     "react-native-maps": "0.25.0",
-    "react-native-music-control": "0.10.6",
+    "react-native-music-control": "0.10.5",
```

Comment:
This is a regression. I guess on purpose?

Reply:
good catch! on codecov and storybook too. I'll upgrade them back to what they were and make sure the app doesn't break.